packages/apps/plugins/plugin-status-bar/src/components/StatusBar.tsx

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React, { forwardRef, type ReactNode } from 'react';
+
+import { type ThemedClassName } from '@dxos/react-ui';
+import { mx } from '@dxos/react-ui-theme';
+
+type StatusBarTextProps = ThemedClassName<{ children: ReactNode }>;
+
+const StatusBarText = forwardRef<HTMLSpanElement, StatusBarTextProps>(({ classNames, children }, forwardedRef) => (
+  <span className={mx(classNames)} ref={forwardedRef}>
+    {children}
+  </span>
+));
+
+type StatusBarButtonProps = ThemedClassName<{ children: ReactNode }> & React.HTMLAttributes<HTMLButtonElement>;
+
+const StatusBarButton = forwardRef<HTMLButtonElement, StatusBarButtonProps>(
+  ({ classNames, children, ...props }, forwardedRef) => (
+    <button
+      className={mx(
+        'flex items-center gap-2 p-1 px-2 rounded-sm select-none hover:bg-gray-200 cursor-pointer active:bg-gray-300',
+        classNames,
+      )}
+      ref={forwardedRef}
+      {...props}
+    >
+      {children}
+    </button>
+  ),
+);
+
+type StatusBarItemProps = ThemedClassName<{ children: ReactNode }>;
+
+const StatusBarItem = forwardRef<HTMLDivElement, StatusBarItemProps>(({ classNames, children }, forwardedRef) => (
+  <div className={mx('flex items-center gap-2 p-1 px-2 rounded-sm select-none', classNames)} ref={forwardedRef}>
```

Comment:
Lots of `div`s here that have no `role`, this will be fairly inaccessible as it is currently.

I suspect the status bar’s content is probably a [`menubar`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menubar_role) with items (some clickable, some not) within.